src/Tiny4kOLED_common.h

```diff
@@ -74,36 +76,51 @@ union DCUnicodeCodepoint {
 
 // Digistump uses
 // # define F(s) ((fstr_t*)PSTR(s))
+#ifdef RP2040
+#include <cstddef>
+#include <string>
+
+#define DATACUTE_F_MACRO_T const char
+using byte = uint8_t;
+#define pgm_read_byte(addr) *reinterpret_cast<const uint8_t *>(addr)
+#define pgm_read_word(addr) *reinterpret_cast<const uint16_t *>(addr)
+#define PGM_P const char *
 
-#ifndef DATACUTE_F_MACRO_T
-#ifdef ARDUINO_AVR_DIGISPARK
-#define DATACUTE_F_MACRO_T fstr_t
 #else
-#define DATACUTE_F_MACRO_T const __FlashStringHelper
-#endif
-#endif
 
-#ifndef FPSTR
-#define FPSTR(pstr_pointer) (reinterpret_cast<DATACUTE_F_MACRO_T *>(pstr_pointer))
-#endif
+# ifndef DATACUTE_F_MACRO_T
+#  ifdef ARDUINO_AVR_DIGISPARK
+#   define DATACUTE_F_MACRO_T fstr_t
+#  else
+#   define DATACUTE_F_MACRO_T const __FlashStringHelper
+#  endif
+# endif
+
+# ifndef FPSTR
+#  define FPSTR(pstr_pointer) (reinterpret_cast<DATACUTE_F_MACRO_T *>(pstr_pointer))
+# endif
+#endif // RP2040
 
 // ----------------------------------------------------------------------------
 
 class SSD1306Device {
 
 	public:
-		SSD1306Device(void (*wireBeginFunc)(void), bool (*wireBeginTransmissionFunc)(void), bool (*wireWriteFunc)(uint8_t byte), uint8_t (*wireEndTransmissionFunc)(void));
+        SSD1306Device(void (*wireBeginFunc)(),
+                      bool (*wireBeginTransmissionFunc)(),
+                      bool (*wireWriteFunc)(uint8_t byte),
+                      uint8_t (*wireEndTransmissionFunc)());
 
-		void begin(void);
-		void begin(uint8_t init_sequence_length, const uint8_t init_sequence []);
+        void begin();
```

Comment:
please change indenting characters back to what they were so that the files are not inconsistent